.gitignore

```diff
@@ -5,3 +5,4 @@ stack.yaml
 cabal.project.local
 .ghci
 public/
+.vscode
```

Comment:
I think we’d prefer to keep editor-specific stuff out of tree; handling your editor’s ephemera is best left to global .gitignore rules rather than per-project.